tests/test_modeling_rag.py

```diff
@@ -848,6 +848,63 @@ def test_rag_sequence_generate_batch(self):
         ]
         self.assertListEqual(outputs, EXPECTED_OUTPUTS)
 
+    @slow
+    def test_rag_sequence_generate_batch_from_context_input_ids(self):
```

Comment:
HI Patrick, since `test_model_generate(self)` check only if generated output is not None . So I add this full generation test instead where `EXPECTED_OUTPUTS` is copied from the `input_ids` counterpart.

Reply:
Yeah makes sense! Could you however also add a "is-not-none" assert for this use case right below the test here: https://github.com/huggingface/transformers/blob/6189ae99603bd5dc14c5631f1b4562f78e24d575/tests/test_modeling_rag.py#L268 - this way the new feature would be checked at every "fast" test as well. 

Think after this, we can merge (I'll take care of `make style`)

Reply:
Understood. `test_model_generate_from_context_input_ids` is added as suggested :)